### DISCUSSION TITLE
what is the difference between the 'netbox-documents' and 'netbox-attachments' plugins?

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#17120] [Q&A]
Curious for people that use one over the other. What is the difference between these 2 plugins? I've read through the documentation of each, and they both appear to do the same thing.
Both are actively developed and have released updates in the last few weeks, so I'm wondering why you would pick one over the other?

NetBox Attachments: https://github.com/Kani999/netbox-attachments
NetBox Documents: https://github.com/jasonyates/netbox-documents

**Comment by Kani999** [comment]
- NetBox Documents only supports the following models:
    - Circuits
    - Devices
    - Device Types
    - Sites
    - Locations
    - Virtual Machines
    - Circuit Providers


On the other hand, NetBox attachments allow you to assign files to any model you want. Additionally, you can enable file assignments to plugin models if they are properly developed, which requires a small change in the HTML template.

**Reply by ghost** [reply]
you might be biased since you wrote netbox-attachments, but would you say that netbox-attachments is therefore more flexible/feature rich since it supports more models than netbox-documents?

**Reply by Kani999** [reply]
- `Documents` maybe more robust and even faster. Documents are always locked to a certain model that is supported. 
- `Attachments` is more flexible (I suppose).
